test/micro/org/openjdk/bench/java/text/DateFormatterBench.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2024, Alibaba Group Holding Limited. All Rights Reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
```

Comment:
Copyright headers for test cases should not include this "classpath exception" clause. Applies to all new added test files.

Reply:
GPL template available here: https://github.com/openjdk/jdk/blob/master/make/data/license-templates/gpl-header